src/lib.rs

```diff
@@ -303,6 +303,8 @@ impl Ext4 {
             })
```

Comment:
Perhaps as a follow-up: augment this error to make it clear when the block index has been altered by the journal?

Reply:
Good idea, filed https://github.com/nicholasbishop/ext4-view-rs/issues/436